### DISCUSSION TITLE
Using "tacotron2-DDC_ph" Model for Male Voice in Coqui TTS

### DISCUSSION THREAD

**Discussion by faryar76** [coqui-ai/TTS#2780] [General Q&A]
Hi everyone,

I hope you're all doing well. I came across the Coqui TTS repository (https://github.com/coqui-ai/TTS) and have been quite impressed with its capabilities. I'm interested in using the "tacotron2-DDC_ph" model for a project, but I have a specific requirement - I need the synthesized voice to be male.

Before I proceed, I wanted to inquire whether it is possible to utilize the "tacotron2-DDC_ph" model to generate speech in a male voice. If so, could someone kindly guide me on how to configure the model accordingly? I've reviewed the documentation and code, but I couldn't find explicit instructions for enabling male voice synthesis with this particular model.

Any insights, guidance, or suggestions on this matter would be greatly appreciated. Thank you all for your time and contributions to this fantastic project!

Best regards,

**Comment by developer-hassan** [comment]
I am having the same issue
